flink-tests/src/test/java/org/apache/flink/test/streaming/runtime/SinkITCase.java

```diff
@@ -235,8 +235,7 @@ public void writerAndGlobalCommitterExecuteInBatchMode() throws Exception {
 
 	private StreamExecutionEnvironment buildStreamEnv() {
 		final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
-		final Configuration config = new Configuration();
```

Comment:
Huh, this was not doing anything, right?

Reply:
No. I will also actually update the method below `buildBatchEnv()`.